### PR TITLE
added validation for new pages

### PR DESCRIPTION
**Pull request by Joe-Cun-Civica**
added validation and enums, all these files were tested locally and are working

### CODE REVIEW

File: enums/businessScaleEnum.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    value: { en: "To provide food directly to other businesses" }
+  },
+  SENIOR_YOUTH: {
+    key: "SENIOR_YOUTH",
+    value: { en: "To those under 5 years old or over 65 years old" }
+  },
+  HEALTHCARE: {
+    key: "HEALTHCARE",
+    value: {
+      en: "To residents in a healthcare setting(for example in a hospital, nursing home or similar)"
```
Comment (simon-says-test) [inline enums/businessScaleEnum.js:31]: missing space before bracket

File: enums/foodTypeEnum.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const foodTypeEnum = {
+  RAW_MEAT: {
```
Comment (simon-says-test) [inline enums/foodTypeEnum.js:2]: I think this key/value has changed at their request - full description for value and amended key value needed. Make sure updated in front-end if required

File: enums/businessScaleEnum.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    }
+  },
+  NONE: {
+    key: "NONE",
+    value: { en: "None" }
```
Comment (simon-says-test) [inline enums/businessScaleEnum.js:36]: "None of these" if we're keeping strictly with their doc

File: enums/foodTypeEnum.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      en: "Food that has been cooked and/or reheated in the establishment"
+    }
+  },
+  IMPORTED: {
+    key: "IMPORTED",
+    value: { en: "Food that your business has imported (from outside the UK)" }
+  },
+  NONE: {
+    key: "NONE",
+    value: { en: "None" }
```
Comment (simon-says-test) [inline enums/foodTypeEnum.js:26]: "None of these" if we're keeping strictly with their doc

File: enums/processingActivitiesEnum.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+      en: "Production of foods of animal origin that are not thoroughly cooked (for example steak tartare, ceviche, carpaccio)"
+    }
+  },
+  REWRAPPING_OR_RELABELLING: {
+    key: "REWRAPPING_OR_RELABELLING",
+    value: { en: "Rewrapping and relabelling previously wrapped food" }
+  },
+  NONE: {
+    key: "NONE",
+    value: { en: "None" }
```
Comment (simon-says-test) [inline enums/processingActivitiesEnum.js:32]: "None of these" if we're keeping strictly with their doc

File: validationFunctions/validateBusinessScale.js [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+  // If 'NONE' or 'DONT_KNOW' are selected, they must be the only selection
+  if ((hasNone || hasDontKnow) && businessScale.length > 1) {
+    return false;
+  }
+
+  // Validate that each selected value is part of the businessScaleEnum
+  for (let activity of businessScale) {
+    if (!businessScaleEnum[activity]) {
+      console.log("val1" + activity);
```
Comment (simon-says-test) [inline validationFunctions/validateBusinessScale.js:35]: remove, along with other console.log below

File: validationFunctions/validateBusinessScale.js [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+  const hasNone = businessScale.includes(businessScaleEnum.NONE.key);
+  const hasDontKnow = businessScale.includes(businessScaleEnum.DONT_KNOW.key);
+
+  // If 'NONE' or 'DONT_KNOW' are selected, they must be the only selection
+  if ((hasNone || hasDontKnow) && businessScale.length > 1) {
+    return false;
+  }
+
+  // Validate that each selected value is part of the businessScaleEnum
+  for (let activity of businessScale) {
```
Comment (simon-says-test) [inline validationFunctions/validateBusinessScale.js:33]: rename "activity". If businessScale array was plural then could rename to businessScale

File: validationFunctions/validateFoodType.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
+/**
+ * Function for validating the food type activities field
+ * @module functions/validateFoodType
+ */
+
+const { foodTypeEnum } = require("../enums/foodTypeEnum");
+
+/**
+ * Runs custom validation on the business scale activities. The function will return true if:
```
Comment (simon-says-test) [inline validationFunctions/validateFoodType.js:9]: not business scale

File: validationFunctions/validateFoodType.js [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+  const hasNone = foodType.includes(foodTypeEnum.NONE.key);
+  const hasDontKnow = foodType.includes(foodTypeEnum.DONT_KNOW.key);
+
+  // If 'NONE' or 'DONT_KNOW' are selected, they must be the only selection
+  if ((hasNone || hasDontKnow) && foodType.length > 1) {
+    return false;
+  }
+
+  // Validate that each selected value is part of the foodTypeEnum
+  for (let activity of foodType) {
```
Comment (simon-says-test) [inline validationFunctions/validateFoodType.js:33]: as per business scale

File: validationFunctions/validateProcessingActivities.js [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+ * Function for validating the business scale activities field
+ * @module functions/validateProcessingActivities
+ */
+
+const {
+  processingActivitiesEnum
+} = require("../enums/processingActivitiesEnum");
+
+/**
+ * Runs custom validation on the business scale activities. The function will return true if:
```
Comment (simon-says-test) [inline validationFunctions/validateProcessingActivities.js:11]: not business scale

File: validationFunctions/validateProcessingActivities.js [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+    processingActivitiesEnum.DONT_KNOW.key
+  );
+
+  // If 'NONE' or 'DONT_KNOW' are selected, they must be the only selection
+  if ((hasNone || hasDontKnow) && processingActivities.length > 1) {
+    return false;
+  }
+
+  // Validate that each selected value is part of the foodTypeEnum
+  for (let activity of processingActivities) {
```
Comment (simon-says-test) [inline validationFunctions/validateProcessingActivities.js:42]: as per business scale

**Review by simon-says-test — CHANGES_REQUESTED** [review]
functionally pretty good. Just a bit of tidying up really.

File: enums/businessScaleEnum.js (hunk 3 above)
Comment (Joe-Cun-Civica) [reply enums/businessScaleEnum.js]: updated to none of these

File: validationFunctions/validateFoodType.js (hunk 9 above)
Comment (simon-says-test) [reply validationFunctions/validateFoodType.js:33]: still to fix

**Review by simon-says-test — APPROVED** [review]

**Pull request merged**